sem_covid/entrypoints/ml_dags/classifiers_dag.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
```

Comment:
This file is not related to SC-381.

Reply:
to update the project and resolve the conflicts, i pushed the file from the main in this branch. Those 100+ files are updated files from the main. And this one is part of them.